A post to officially announce the Development Fund. (#703)
godotengine/godot-website: collections/_article/godot-developer-fund.md

---
title: "Introducing the new Godot Development Fund"
excerpt: "A direct way to support the development of Godot."
categories: ["news"]
author: Emi
image: /storage/blog/covers/developer-fund.jpg
date: 2023-09-12 11:51:10
---

We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine - the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us keep the project alive.
 

> ## Reducing Platform Fees, Amplifying Impact

Reviewer: I think we normally use "Sentence case" for headings, not "Title Case".
```suggestion
## Reducing platform fees, amplifying impact
```

Author: That is correct! Thanks!

> We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine - the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us keep the project alive.

Reviewer 2: ```suggestion
We are excited to introduce a major improvement in  funding the development efforts of the Godot Engine - the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us secure stable funding and ensure the longevity of the project.
```

I'm not sure my suggestion is perfect. But I do think we should use more subtle language than "help us keep the project alive". While technically it has the same meaning as my suggestion, it also makes it sound like the project is out of funds (which was a common misconception from our last funding post). 

Personally, I think we should be highlighting 2 things:
1. Longevity (we need to ensure that development pace is maintained as long as possible)
2. Independence. The more funding we have from individuals, the less risk there is that Godot is "selling out". This is more of a public perception thing than an actual problem.

Reviewer 2: Ah, I see you do that more below.

> We are excited to introduce a major improvement in  funding the development efforts of the Godot Engine - the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us secure stable funding and ensure the longevity of the project.

Reviewer 3: ```suggestion
We are excited to introduce a major improvement in funding the development efforts of the Godot Engine – the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us secure stable funding and ensure the longevity of the project.
```

>  

Reviewer 3: ```suggestion
 
 The Development Fund was [soft-launched in July](/article/funding-breakdown-and-hiring-process/) and already welcome a lot of new and old sponsors, but we need to take it to the next level to sustain the current development pace, and increase it with new hires.
```

> The Development Fund will make The Godot Project more sustainable and independent. This new platform lets us significantly reduce platform fees, so your contributions go directly towards the development and improvement of the Godot Engine itself.

Reviewer 3: ```suggestion
The Development Fund will make the Godot Project more sustainable and independent. This new platform lets us significantly reduce platform fees, so your contributions go directly towards the development and improvement of the Godot Engine itself.
```

> The Development Fund will make The Godot Project more sustainable and independent. This new platform lets us significantly reduce platform fees, so your contributions go directly towards the development and improvement of the Godot Engine itself.

Reviewer 3: Should we clarify what platform fees mean here? It's not just about cutting the middleman, Patreon is a platform for commercial subscriptions and thus they charge VAT. So it's not just their 5-8% cut + 5% payment processor, but an additional +20-25% VAT for EU people.

Author: I didn't have the numbers exactly about how much Patreon got, so I wasn't sure about adding hard numbers. If those are the percentages we could add it 100%

Reviewer 3: Well I wouldn't try to give clear numbers either, but I think detailing that we're saving on both platform fees and, where applicable, VAT, would be useful.

If you are currently supporting us [through Patreon](https://www.patreon.com/godotengine), we encourage you to switch to the [Godot Development Fund](https://fund.godotengine.org) to increase the effectiveness of your donations.

> ## Giving Back to the Community

Reviewer: ```suggestion
## Giving back to the community
```

> The support of the community is what keeps making it possible to reach [new heights](https://godotengine.org/article/godot-4-0-sets-sail/).With you, we can achieve even more!

Reviewer: ```suggestion
The support of the community is what keeps making it possible to reach [new heights](https://godotengine.org/article/godot-4-0-sets-sail/). With you, we can achieve even more!
```

Every donation, no matter the size, helps us immensely! With your help we can hire more developers, create high quality demos, pay for hosting services and infrastructure, and make the best possible Godot Engine. The more funds we have, the more we can give back to you and the entire gamedev community.

> ## Future Enhancements

Reviewer: ```suggestion
## Future enhancements
```

While this is a good, fully functioning starting point for the platform, we will keep improving the Developer Fund.

> Moving forward, the platform will give automated rewards, like a special Discord role, and you'll be able to showcase your contribution level next to your username on our main platforms. 

Reviewer 2: ```suggestion
In the future, the platform will give automated rewards, like a special Discord role, and you'll be able to showcase your contribution level next to your username on our main platforms. 
```

> In the future, the platform will give automated rewards, like a special Discord role, and you'll be able to showcase your contribution level next to your username on our main platforms. 

Reviewer 3: ```suggestion
In the future, the platform will give automated rewards, like a special Discord role, and you'll be able to showcase your contribution level next to your username on our main platforms.
```

## Freedom and independence

By relying more on user-funded contributions, we reduce our dependence on large grants from corporations. This financial independence empowers us to prioritize the needs and interests of our community and the open-source principles at the core of our projects. Your help ensures that we can create our own journey, uphold our core values, and continue to evolve the engine in ways that truly benefit you, the users.

> ## A Heartfelt Thanks

Reviewer: ```suggestion
## A heartfelt thanks
```

> By relying more on user-funded contributions, we reduce our dependence on large grants from corporations. This financial independence empowers us to prioritize the needs and interests of our community and the open-source principles at the core of our projects. Your help ensures that we can create our own journey, uphold our core values, and continue to evolve the engine in ways that truly benefit you, the users.

Reviewer 3: ```suggestion
By relying more on recurring user-funded contributions and company sponsorships, we reduce our dependence on large one-time grants from corporations. This financial independence empowers us to prioritize the needs and interests of our community and the open source principles at the core of our projects. Your help ensures that we can create our own journey, uphold our core values, and continue to evolve the engine in ways that truly benefit you, the users.
```

Maybe helps address @clayjohn's concern? We do need to encourage for corporate sponsorships _a lot_ more, which are usually upheld for multiple years and help us plan ahead.

Author: yeah, I like this edit

> By relying more on user-funded contributions, we reduce our dependence on large grants from corporations. This financial independence empowers us to prioritize the needs and interests of our community and the open-source principles at the core of our projects. Your help ensures that we can create our own journey, uphold our core values, and continue to evolve the engine in ways that truly benefit you, the users.

Reviewer 4: you, the users -> you and the rest of the Godot community

(perhaps?)

---
title: "Introducing the new Godot Development Fund"
excerpt: "We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine."
categories: ["news"]
author: Emi
image: /storage/blog/covers/developer-fund.jpg
date: 2023-09-12 11:51:10
---

We are excited to introduce a major improvement in funding the development efforts of the Godot Engine – the [Godot Development Fund](https://fund.godotengine.org)! The goal for the Development Fund is to create a direct way for everyone to help us secure stable funding and ensure the longevity of the project.
 

>  The Development Fund was [soft-launched in July](/article/funding-breakdown-and-hiring-process/) and already welcome a lot of new and old sponsors, but we need to take it to the next level to sustain the current development pace, and increase it with new hires.

Reviewer 3: ```suggestion
The Development Fund was [soft-launched in July](/article/funding-breakdown-and-hiring-process/) and already welcomed a lot of new and old sponsors, but we need to take it to the next level to sustain the current development pace, and increase it with new hires.
```

## Reducing platform fees, amplifying impact

> The Development Fund will make the Godot Project more sustainable and independent. This new platform lets us significantly reduce platform fees, so your contributions go directly towards the development and improvement of the Godot Engine itself.

Reviewer 3: ```suggestion
The Development Fund will make the Godot Project more sustainable and independent. This new platform lets us significantly reduce platform fees, and as a non-profit we do not charge VAT on donations, unlike a platform like Patreon does. So your contributions go directly towards the development and improvement of the Godot Engine itself.
```

Not super happy with the phrasing, but I think it's worth mentioning.
